### PR TITLE
Build navbar

### PR DESCRIPTION
**Pull request by matthewcotton**
Create Navbar and add to App.js (so that it is visible on all pages).

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <Link to="/">
+              <p>Home</p>
+            </Link>
+          </Navbar.Text>
+          <NavDropdown title="Locations" id="basic-nav-dropdown">
+            <NavDropdown.Item href="/restaurants/antarctic">Antarctic</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/arctic">Arctic</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/sahara">Sahara</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/arabian">Arabian</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/gobi">Gobi</NavDropdown.Item>
```
Comment (MikeDThorpe) [inline src/components/Navbar.js]: file paths may require a period at the start - ./

Comment (matthewcotton) [reply src/components/Navbar.js]: Adding a "." caused router link issues due to the nested routes. So this has not been changed.

Comment (matthewcotton) [reply src/components/Navbar.js]: Using href was causing a page refresh so I am using LinkContainer from the newly added package react-router-bootstrap.

**Comment by matthewcotton** [comment]
The placement of the Navbar in App.js was blocking all other page content from being shown therefore I have moved this to fix the issue. This also enabled the nested routing for the restro's to work so I have uncommented this.

**Pull request merged**